### PR TITLE
add box deletion

### PR DESCRIPTION
**Pull request by maxvozeler**

### CODE REVIEW

File: jiffybox/api.py [hunk 1]
```diff
@@ -357,6 +363,15 @@ def create_box(self, *args, **kwargs):
         """
         return Box.create(self, *args, **kwargs)
 
+    def delete_box(self, id):
```
Comment (t-8ch) [inline jiffybox/api.py:366]: This will not work

Comment (maxvozeler) [inline jiffybox/api.py]: Updated pull request, also adds a confirmation prompt for delete

**Comment by maxvozeler** [comment]
Updated pull request, also adds a confirmation prompt for delete

**Comment by t-8ch** [comment]
:sparkles: :100: :sparkles:

**Pull request merged**